translate/storage/placeables/__init__.py

```diff
@@ -45,14 +45,14 @@
 Please refer to the XLIFF specification to get a better understanding.
 """
 
-import base
-import interfaces
-import general
-import xliff
-from base import *
-from base import __all__ as all_your_base
-from strelem import StringElem
-from parse import parse
+from . import base
+from . import interfaces
+from . import general
+from . import xliff
```

Comment:
@claudep Lets be cautious and leave that cleanup out. If you rebase this PR I will merge it.